All EIPIP Meetings/Meeting 049.md

```diff
@@ -0,0 +1,253 @@
+# EIPIP Meeting 49 Notes
+
+## Meeting Date/Time: Wednesday, Feb 09, 2022, at 15:00 UTC
+
+### Meeting Duration: 40 minutes, 36 seconds
+
+### [GitHub Agenda Page](https://github.com/ethereum-cat-herders/EIPIP/issues/107)
+
+### [Audio/Video of the meeting](https://youtu.be/ATlQApSKDeM)
+
+### Moderator: Pooja Ranjan
+
+### Notes: Shashank
+
+----
+
+## ACTION ITEMS
+
+**ACTION 49.1**:
+
+**ACTION 49.2**:
+
+**ACTION 49.3**:
+
```

Comment:
```suggestion
**ACTION/DECISION 49.4**: Formalize a process for EIP standardization in EIPIP meetings. When the process is well defined, it can be shared in the ACD meeting, so people provide feedback. 
Progress on Execution specs be continued to be discussed in EIPIP meetings.

**ACTION/DECISION 49.5**: Pooja will follow with FEM team on limited editing window on FEM thread.  


```